stoplight.js

```diff
@@ -0,0 +1,8 @@
+  $(".bulb").on("light:toggle", function(e) {
+    var stoplight = $(this);
+    if(light.is(".on")) {
+      
+    }
+
```

Comment:
would like to see you come back to this and complete it whenever you get the chance.